bdb/file.c

```diff
@@ -7101,8 +7114,8 @@ uint64_t bdb_queue_size(bdb_state_type *bdb_state, unsigned *num_extents)
 
     *num_extents = 0;
 
-    if (bdb_state->bdbtype != BDBTYPE_QUEUE)
-        return 0;
+    if (bdb_state->bdbtype == BDBTYPE_QUEUEDB)
```

Comment:
This piece of code assumes that bdbtype here can be only  BDBTYPE_QUEUEDB and BDBTYPE_QUEUE. I would put an assert here for that.
One instance of this func being called is glue.c:5419, and the if clause for that is wrong -- if you can please fix with this pr since these are related.

Reply:
I'm more worried about the fact that calc_table_size and calc_table_size_analyze seem top do the same exact thing.

Reply:
Ah, calc_table_size_analyze skips blob sizes, which makes sense.

Reply:
Added assert, removed copy pasta.